pkg/controller/controller.go

```diff
@@ -630,6 +632,67 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 			cont.opflexDeviceDeleted(dn)
 		})
 	go cont.apicConn.Run(stopCh)
```

Comment:
As a follow up patch we might want to consider running this function based on a config. By default the config is enabled, but we can disable it if required.